src/commcare_cloud/terraform/staging/terraform.tf

```diff
@@ -27,57 +26,57 @@ module "commcarehq" {
     {
       server_name           = "web0-staging"
       server_instance_type  = "t2.micro"
-      subnet_index          = 0
+      subnet                = "private-a"
     },
     {
       server_name           = "web1-staging"
       server_instance_type  = "t2.micro"
-      subnet_index          = 1
+      subnet                = "private-b"
```

Comment:
why is this and then next two on different subnets?

Reply:
Yeah so I think there's a large amount of arbitrariness here, but I'm just following the setup G2 left us with when I don't have a specific reason to change it. The only part that's not arbitrary and does serve a purpose, as far as I can tell, is that web0 and web1 are in different _availability zones_ (zones a & b). Availability zones are structured to be two completely parallel systems within the same data center so that a system-wide outage can only possibly affect a single availability zone. We don't have enough redundancy make the application robust against complete outage of a single AZ, but with more machines (esp. redundant proxy) you can get closer to that.

That said, there's no downside to having machines on different AZs, so even if it doesn't fully protect us, it could conceivably be helpful in the case of partial service interruption in a single AZ, and it doesn't cost us anything.